wdl-engine/src/outputs.rs

```diff
@@ -3,20 +3,21 @@
 use std::cmp::Ordering;
 
 use indexmap::IndexMap;
+use serde::Serialize;
 
 use crate::Scope;
 use crate::Value;
 
 /// Represents outputs of a WDL workflow or task.
-#[derive(Default, Debug, Clone)]
+#[derive(Default, Debug, Clone, Serialize)]
```

Comment:
Rather than implement `Serialize` here, I think we should move the `serialize` function which is no longer needed since #277 and turn it into an explicit `Serialize` implementation.